server/src/main/java/org/hyperskill/community/flashcards/card/model/MultipleChoiceQuiz.java

```diff
@@ -0,0 +1,34 @@
+package org.hyperskill.community.flashcards.card.model;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.data.annotation.TypeAlias;
+
+import java.util.List;
+
+/**
+ * Represents a multiple choice quiz card, i.e. where the user is provided with
+ * multiple options of which at least one is correct. Commonly visualized as
+ * having options with checkboxes
+ */
+@Getter
+@Setter
+@ToString(callSuper = true)
+@TypeAlias("MultipleChoiceQuiz")
+public class MultipleChoiceQuiz extends Card {
+    private List<String> options;
+    private List<Integer> correctOptions;
+
```

Comment:
is it possible to use Java records here - or does this come with cons?

Reply:
Records cannot extend classes and we have quite a few shared properties. Having a sealed interface is less beneficial in my opinion.